core/src/main/scala/org/apache/spark/scheduler/BlacklistTracker.scala

```diff
@@ -146,21 +146,31 @@ private[scheduler] class BlacklistTracker (
     nextExpiryTime = math.min(execMinExpiry, nodeMinExpiry)
   }
 
+  private def killExecutor(exec: String, msg: String): Unit = {
+    allocationClient match {
+      case Some(a) =>
+        logInfo(msg)
+        a.killExecutors(Seq(exec), adjustTargetNumExecutors = false, countFailures = false,
+          force = true)
+      case None =>
+        logWarning(s"Not attempting to kill blacklisted executor id $exec " +
```

Comment:
this should probably be info message since its not really anything bad they just aren't using dynamic allocation.